notification.c

```diff
@@ -286,3 +286,51 @@ void notification_handle_button(struct mako_notification *notif, uint32_t button
 		break;
 	}
 }
+
+/*
+ * Searches through the notifications list and returns the next position at which to insert.
+ * If no results for the specified urgency are found, it will return the closest link searching in the direction specifed. (-1 for lower, 1 or upper).
+ */
+static struct wl_list *get_last_notif_by_urgency(struct wl_list *notifications, enum mako_notification_urgency urgency, int direction) {
+	enum mako_notification_urgency current = urgency;
+
+	if (wl_list_length(notifications) == 0) {
```

Comment:
It's better to use `wl_list_empty` instead - it doesn't require to iterate over the whole list

Reply:
wasn't aware of it. thank you!